ci-operator/step-registry/openshift/e2e/test/netobserv-qe/openshift-e2e-test-netobserv-qe-chain.yaml

```diff
@@ -16,6 +16,9 @@ chain:
   - name: CYPRESS_NO_COMMAND_LOG
     default: "1"
     documentation: "disable displaying test commands, assertions, and statuses in the Cypress Test Runner, may cause performance issues resulting in slower tests or the browser crashing"
+  - name: TEST_TIMEOUT
+    default: "35"
+    documentation: it means the duration of one case execution in minutes.
```

Comment:
Can be changed to `Time-out duration for each test-case execution in minutes` or something along those lines